playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_repo.yml

```diff
@@ -0,0 +1,34 @@
+- name: set latest tag from git tag
```

Comment:
Again troys instructions for origin this was missing from producing versioned rpms we call tito in a separate task